runtime-modules/council/src/lib.rs

```diff
@@ -807,21 +837,24 @@ impl<T: Trait> EnsureChecks<T> {
         council_user_id: &T::CouncilUserId,
     ) -> Result<T::AccountId, Error<T>> {
         // ensure user's membership
-        let account_id = Self::ensure_user_membership(origin, council_user_id)?;
+        Self::ensure_user_membership(origin, council_user_id)?;
+
+        // ensure stake is unlockable
+        let staking_account_id = match Self::ensure_previous_stake_is_reusable(council_user_id)? {
```

Comment:
Please split this up into two separate steps for clarity.

Reply:
Missing response.

Reply:
The function has been removed by 'candidacy unstaking' changes https://github.com/Joystream/joystream/pull/1676#discussion_r521504113